pandas/tests/frame/test_analytics.py

```diff
@@ -130,6 +130,11 @@ def test_corr_cov_independent_index_column(self):
             assert result.index is not result.columns
             assert result.index.equals(result.columns)
 
+    def test_corr_invalid_method(self):
+        df = pd.DataFrame(np.random.normal(size=(10, 2)))
+        with pytest.raises(ValueError):
```

Comment:
can you check this with an assert_raises_regex (e.g. match the error)

add the gh issue number as a comment

Reply:
Will do.  Is the convention here to use the most specific regex that matches the error message, or just one that is reasonably specific?

Reply:
reasonable is fine